draft-ietf-quic-http.md

```diff
@@ -1595,9 +1595,10 @@ HTTP_NO_ERROR (0x00):
 : No error.  This is used when the connection or stream needs to be closed, but
   there is no error to signal.
 
-HTTP_WRONG_SETTING_DIRECTION (0x01):
-: A client-only setting was sent by a server, or a server-only setting by a
-  client.
+HTTP_SETTINGS_ERROR (0x01):
+: An endpoint detected an error with a SETTINGS frame: a duplicate setting was
+  detected, a client-only setting was sent by a server, or a server-only setting
+  by a client.
```

Comment:
The suggestion to move HTTP_GENERAL_PROTOCOL_ERROR to 0x01 might be worth taking at the same time, since people are going to change which error to use for these anyway.  (Also, I just noticed that GPE is missing from the IANA table.)

Reply:
That was my initial intention but I thought it might make life harder. However, life is hard regardless so I'll try it out.

> (Also, I just noticed that GPE is missing from the IANA table.)

You reminded me that I fixed this on the MALFORMED_FRAME PR. It makes sense to fix that here though while shuffling things around.

Reply:
Thanks, fixed.